### ISSUE TITLE
Set up NetworkTables to enable communication between RPi and Rio.

### ISSUE THREAD

**Issue by CJKeller03** [Stellar-Robotics/Robot-Code-2024#4]
Add code to both the vision and robot scripts to enable the RPi to stream AprilTag data to the Rio.

**Comment by StellarRobotics** [comment]
Putting this here for future reference as it may be useful:

https://robotpy.readthedocs.io/projects/pynetworktables/en/stable/api.html

**Comment by AbstractestThought** [comment]
This issue should be closed, correct?

**Comment by CJKeller03** [comment]
Not quite yet- we got the code working on the RPi side, but not the robot side yet. Gabe and I actually discovered NetworkTables uses a new protocol this year which is why all the old ntcore example code you guys were trying to run before didn't work 🤦‍♂️

We updated the vision script to use the new ntcore stuff and got it working properly (so the robotpy_networktables lib isn't needed anymore), but have to finish writing the bot side of it.

Right now, that work is happening in the old apriltags branch, but it should probably be moved to its own branch. Maybe we can clean that up and finish it tonight.
